Translation corrections (#415)
07th-mod/higurashi-console-arcs: Update/ura_003.txt

//r珠子はばかばかしい、とため息交じりに肩をすくめて自分の机に戻り、下敷きをうちわ代わりにして涼みを取る。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "珠子はばかばかしい、とため息交じりに肩をすくめて自分の机に戻り、下敷きをうちわ代わりにして涼みを取る。",

> 			NULL, "Tamako shrugged her shoulders and let out an exasperated sigh, then went back to her desk and used the desk pad in place of a fan to cool off.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Tamako shrugged her shoulders and let out an exasperated sigh, then went back to her desk and used her desk pad like a fan to cool off.", Line_Normal);
```

//r…まぁ、それでもこれを借りた時には、相当無理を言ってしまったのも確かだ。さすがにこれまでの貸し借りを差っ引いても、アイス程度では申し訳が立たないだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…まぁ、それでもこれを借りた時には、相当無理を言ってしまったのも確かだ。さすがにこれまでの貸し借りを差っ引いても、アイス程度では申し訳が立たないだろう。",

> 			NULL, "...Well, back when I told her I was borrowing it, I'm sure she did say it was unreasonable. I'm sure that even if you subtracted all the loans and debt I'd accumulated, I wouldn't be able to make it up to her with merely ice cream.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Well, back when I told her I was borrowing it, I'm sure she did say it was unreasonable. I'm sure that even if you subtracted all the loans and debt I'd accumulated, it would take more than just ice cream to pay her back.", Line_Normal);
```

//rまったく…気がないふりをしているくせにこんなひっかけでボロが出るんだから、ほんとバカ正直というかなんというか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まったく…気がないふりをしているくせにこんなひっかけでボロが出るんだから、ほんとバカ正直というかなんというか。",

> 			NULL, "Geez... He pretends not to notice and yet gives up the ghost when he gets caught off guard like this. He's honest to a fault.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Geez... He pretends not to notice, but he gives up the ghost when he gets caught off guard like this. He's honest to a fault.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s14/37/vt4bb_chisa1084", 256, TRUE);
	OutputLine(NULL, "「い、いったいなぁ～。",

> 			NULL, "\"Th-that hurt~", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Th-that hurt~.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 38, "ps3/s14/38/vt4bb_tama1040", 256, TRUE);
	OutputLine(NULL, "「はいパチもんお嬢様、イメージ壊れるから少し黙ってて」",

> 			NULL, "\"Hey, imitation of a refined young lady, you're breaking that image, so be quiet for a bit.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Hey, you imitation of a refined young lady, you're breaking that image, so be quiet for a bit.\"", Line_Normal);
```

//rあぁ、やっぱりこいつはこういう場面でダメダメだねぇ。一言『いい』って伝えるだけで十分好感度アップなのに。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あぁ、やっぱりこいつはこういう場面でダメダメだねぇ。一言『いい』って伝えるだけで十分好感度アップなのに。",

> 			NULL, "Oh boy, that one  really is completely useless in situations like this, isn't he? Even though it would be more than enough to just say \"it's nice.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "Oh boy, that one really is completely useless in situations like this, isn't he? Even though it would be more than enough to just say \"it's nice.\"", Line_Normal);
```
